Source/GoogleMapsCarbonFootprint.js

```diff
@@ -18,7 +18,12 @@ if (href.match(/maps/gi)) {
 
     travelRate = response.travelRate.value;
     displayTravelCost = response.travelRate.displayTravelCost;
-    carbonEmission = response.emissionRate.value;
+    
+      window.carbonEmission1 = response["emissionRate"]["value"];
+
+      window.carbonEmission2 = response["emissionRate"]["value1"];
```

Comment:
1, 2 and 3 are not informative enough. Give more informative names to your variables. For instance, ```carbonEmissionBelow30``` for carbon emission at speeds below 30km/h.